File: point_rotator.java
public class point_rotator {
    public static void main(String[] args) {

    }
    public static double radToDeg(double theta) {
        theta /= Math.PI;
        theta *= 180;
        return theta;
    }
    public static double degToRad(double theta) {
        theta /= 180;
        theta *= Math.PI;
        return theta;
    }
    public static double[] rotatePoint(double x, double y, double theta) {
        double newX = x*Math.cos(theta) + y*Math.sin(theta);
        double newY = x*Math.sin(theta) + y*Math.cos(theta);
        return new double[]{newX, newY};
    }
}

File: point_rotator.py
import math

def rotate_point(x, y, theta) -> tuple: #Function definition with explicit type returning
    """Rotates (x, y) by theta radians"""
    new_x = x*math.cos(theta) + y*math.sin(theta)
    new_y = y*math.cos(theta) + x*math.sin(theta)
    return (new_x, new_y)

def rad_to_deg(theta: float):
    """Converts radians to degrees"""
    theta /= math.pi
    theta *= 180
    return theta

def deg_to_rad(theta: float):
    """Converts degrees to radians"""
    theta /=180
    theta *= math.pi
    return theta

print(deg_to_rad(90))
print(math.pi/2)
print(rotate_point(1, 0, deg_to_rad(90)))